### ISSUE TITLE
postsynaptic early endosome

### ISSUE THREAD

**Issue by dosumis** [geneontology/synapse#25]
The early endosome (EE), characterised by the presence of rab5 and EEA1, acts as the major sorting station on the endocytic pathway. PMID:24727350
Example; RAB5A, EEA1 @PMID:24727350/19603039

**Comment by gillespm** [comment]
"Postsynaptic cell..." maybe? Just trying to avoid assumptions.

Marc


M.E. Gillespie PhD
Professor
Saint John's University
Queens, NY USA
http://gillespielab.org

On Aug 10, 2015, at 6:31 AM, David Osumi-Sutherland <notifications@github.com<mailto:notifications@github.com>> wrote:


The early endosome (EE), characterised by the presence of rab5 and EEA1, acts as the major sorting station on the endocytic pathway. PMID:24727350
Example; RAB5A, EEA1 @PMID<https://github.com/PMID>:24727350/19603039

**Comment by dosumis** [comment]
> "Postsynaptic cell..." maybe? Just trying to avoid assumptions.

Yep agree.  This was a copy and paste from the doc Frank has been working on.  

Have added:

label "postsynaptic early endosome"@en
has_obo_namespace "cellular_component"
definition "An early endosome of the postsynapse. It acts as the major sorting station on the endocytic pathway, targetting neurotransmitter receptors for degregation or recycling."
comment "Commonly used markers for postsynaptic early endosomes include RAB5A and EEA1."
EquivalentTo: 'early endosome' and ('part of' some postsynapse)
